modules/conversion-center/base/widget-link-in-bio-base.php

```diff
@@ -1426,12 +1426,19 @@ protected function add_style_background_controls(): void {
 				'label' => esc_html__( 'Opacity', 'elementor' ),
 				'type' => Controls_Manager::SLIDER,
 				'range' => [
-					'px' => [
+					'%' => [
 						'max' => 1,
 						'min' => 0.10,
 						'step' => 0.01,
 					],
 				],
+				'default' => [
+					'unit' => '%',
+					'size' => 0.5,
+				],
```

Comment:
@willhowat Can the default be in the CSS only?

Reply:
Hi @louiswol94 the request was to add these values to the controls. In this case I've aligned the default in the CSS as well